source/CardGames.py

```diff
@@ -224,15 +207,31 @@ def start_game(self):
     dealer.dealCards(5, self.players)
     for player in self.players:
       player.showHand(player.hand)
-    
+
+  def check_if_empty(self):
+    if len(self.info_player_asked) == 0:
+          self.game_over()
+
+  def game_over(self):
```

Comment:
This end-of-game function is very cool maybe in the future we can make it more complicated.

Reply:
Thanks! Yes, I want to design it a little bit better, but this is what I had for now. Thanks for the feedback.